src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -511,11 +511,6 @@ function calc_boundary_flux_by_direction_divergence!(surface_flux_values::Abstra
     return nothing
 end
 
-# `cache` is the hyperbolic cache, i.e., in particular not `cache_parabolic`.
-# This is because mortar handling is done in the (hyperbolic) `cache`.
-# Specialization `flux_viscous::Vector{Array{uEltype, 4}}` needed since 
-#`prolong2mortars!` in dg_2d.jl is used for both purely hyperbolic and 
-# hyperbolic-parabolic systems.
 function prolong2mortars!(cache, flux_viscous::Vector{Array{uEltype, 4}},
```

Comment:
```suggestion
# `cache` is the hyperbolic cache, i.e., in particular not `cache_parabolic`.
# This is because mortar handling is done in the (hyperbolic) `cache`.
# Specialization `flux_viscous::Vector{Array{uEltype, 4}}` needed since 
#`prolong2mortars!` in dg_2d.jl is used for both purely hyperbolic and 
# hyperbolic-parabolic systems.
function prolong2mortars!(cache, flux_viscous::Vector{Array{uEltype, 4}},
```